test/hca_metadata_api/test.py

```diff
@@ -754,6 +760,34 @@ def test_name_substitution(self):
             self.assertEqual(expected_slash_after, with_slash_after)
 
 
+class TestSchema(AzulUnitTestCase):
+
+    def json_body(self, domain: str) -> JSON:
+        return {
+            'content': {
+                'describedBy': f'https://{domain}/type/protocol/7.1.0/protocol',
+                'protocol_core': {
+                    'protocol_id': 'test-protocol'
+                },
+                'schema_type': 'protocol'
+            },
+            'hca_ingest': {
+                'document_id': str(uuid4()),
```

Comment:
```suggestion
                'document_id': str(uuid4()),
```

Why did you decide not to hard-code the UUID?

Reply:
To make it clear that this is a test entity, not from any canned/live bundle.

Reply:
I wouldn't see that and come to the conclusion you intended.

We want our tests to be reproducible. Generating a v4 UUID adds randomness. 

Even if we use randomness, say, in the IT, we log the random generator's seed so that we can still reproduce a test, despite the randomness. Besides, if I wanted to check if the hard-coded value relates to a can, I can always do a text search. And we never tracked whether a UUID relates to a live bundle. Most cans come from live bundles, but then we modified the cans so I wouldn't be surprised to see a few new UUIDs in them, either.

Please allocate a new UUID and hard-code it here.